crates/ruff_python_trivia/src/tokenizer.rs

```diff
@@ -71,23 +71,18 @@ pub fn lines_after(offset: TextSize, code: &str) -> u32 {
 }
 
 /// Returns the position after skipping any trailing trivia up to, but not including the newline character.
```

Comment:
The docstring is kinda confusing, do we always know that there is no none-trivia before the newline, and what if there is?

Reply:
Haha didn't you write this!??

Reply:
git blame says i'm innocent (but it could as well have been mine)